### PR TITLE
Add etcd-client-urls flag in backend reference

### LINKED ISSUE #2005: Add the new flag `--etcd-client-urls` to backend reference
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [X] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Expected Behavior
For 5.16, there is a new flag called `--etcd-client-urls`. This is used when a Sensu backend is not operating as an etcd member.

## Current Behavior
Missing new content.

## Possible Solution
Add that content to backend reference.

## Affected Docs Pages
https://docs.sensu.io/sensu-go/5.15/reference/backend/

## Steps to Reproduce (for bugs)

1.
2.
3.

## Context

* Browser(s) and version(s): 
* OS:

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Add flag `--etcd-client-urls` in backend reference.

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/2005

## Review Instructions
Need to confirm I described the flag correctly in the list and table.

### CODE REVIEW

File: content/sensu-go/5.16/reference/backend.md [hunk 1]
```diff
@@ -639,6 +640,22 @@ sensu-backend start --etcd-client-cert-auth
 etcd-client-cert-auth: true{{< /highlight >}}
 
 
+| etcd-client-urls      |      |
+------------------------|------
+description             | List of URLs to listen on for client traffic to use when a sensu-backend is not operating as an etcd member. Use with sensu-backend and [sensu-backend init][20]. Returns a client that connects to an external etcd. If you do not use this flag, sensu-backend and sensu-backend-init will fall back to [--etcd-listen-client-urls][21].
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/reference/backend.md]:
```suggestion
description             | List of URLs to listen on for client traffic when a sensu-backend is not operating as an etcd member. Use with sensu-backend and [sensu-backend init][20]. Returns a client that connects to an external etcd. If you do not use this flag, sensu-backend and sensu-backend-init will fall back to [--etcd-listen-client-urls][21].
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]
I don't think the flag description accurately represents what this flag does.

For context, sensu-backend daemons need to talk to an etcd database, whether that's the embedded instance which runs inside sensu-backend by default, or an external instance running on a distinct host.

My understanding is that this flag provides the sensu-backend with a list of external etcd database URLs which the backend will connect to as a client when running with the `--no-embed-etcd` flag.

In my experience, a daemon which is said to be listening for traffic on a URL is binding a socket of its own to that ip/port combination. As a result, when I read the current description, it seems to be telling me that sensu-backend will bind a socket to these URLs and listen for etcd client traffic, which would not be useful if the embedded etcd isn't running ("not operating as an etcd member")

File: content/sensu-go/5.16/reference/backend.md [hunk 2]
```diff
@@ -246,6 +246,7 @@ Store Flags:
       --etcd-advertise-client-urls strings         list of this member's client URLs to advertise to the rest of the cluster. (default [http://localhost:2379])
       --etcd-cert-file string                      path to the client server TLS cert file
       --etcd-cipher-suites strings                 list of ciphers to use for etcd TLS configuration
+      --etcd-client-urls                           list of urls to listen on for client traffic when sensu-backend is not operating as an etcd member
```
Comment (cwjohnston) [inline content/sensu-go/5.16/reference/backend.md]:
```suggestion
      --etcd-client-urls string                    client URLs to use when operating as an etcd client
```

File: content/sensu-go/5.16/reference/backend.md [hunk 3]
```diff
@@ -639,6 +640,22 @@ sensu-backend start --etcd-client-cert-auth
 etcd-client-cert-auth: true{{< /highlight >}}
 
 
+| etcd-client-urls      |      |
+------------------------|------
+description             | List of URLs to listen on for client traffic when a sensu-backend is not operating as an etcd member. Use with sensu-backend and [sensu-backend init][20]. Returns a client that connects to an external etcd. If you do not use this flag, sensu-backend and sensu-backend-init will fall back to [--etcd-listen-client-urls][21].
```
Comment (cwjohnston) [inline content/sensu-go/5.16/reference/backend.md]:
```suggestion
description             | List of client URLs to use when a sensu-backend is not operating as an etcd member. To configure sensu-backend for use with an external etcd instance, use this flag in conjunction with `--no-embed-etcd` when executing sensu-backend start or [sensu-backend init][20] . If you do not use this flag when using `--no-embed-etcd`, sensu-backend start and sensu-backend-init will fall back to [--etcd-listen-client-urls][21].
```

**Comment by hillaryfraley** [comment]
@cwjohnston thank you! I felt like I wasn't quite getting it.

**Review by cwjohnston — COMMENTED** [review]
Looks good to me, but I'd like @echlebek to give the final approval.

**Review by echlebek — APPROVED** [review]
Perfect!

**Review by cwjohnston — APPROVED** [review]

**Pull request merged**